src/scenes/DeliverablesRouter/SpeciesDeliverableView.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useMemo } from 'react';
+
+import { Box, Typography, useTheme } from '@mui/material';
+import { BusySpinner } from '@terraware/web-components';
+
+import { Crumb } from 'src/components/BreadCrumbs';
+import Metadata from 'src/components/DeliverableView/Metadata';
+import MobileMessage from 'src/components/DeliverableView/MobileMessage';
+import TitleBar from 'src/components/DeliverableView/TitleBar';
+import { EditProps, ViewProps } from 'src/components/DeliverableView/types';
+import Page from 'src/components/Page';
+import Card from 'src/components/common/Card';
+import Button from 'src/components/common/button/Button';
+import { APP_PATHS } from 'src/constants';
+import { useLocalization } from 'src/providers';
+import { useParticipantData } from 'src/providers/Participant/ParticipantContext';
+import { requestListParticipantProjectSpecies } from 'src/redux/features/participantProjectSpecies/participantProjectSpeciesAsyncThunks';
+import { selectParticipantProjectSpeciesListRequest } from 'src/redux/features/participantProjectSpecies/participantProjectSpeciesSelectors';
+import { useAppDispatch, useAppSelector } from 'src/redux/store';
+import RejectedDeliverableMessage from 'src/scenes/DeliverablesRouter/RejectedDeliverableMessage';
+import strings from 'src/strings';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+
+export type Props = EditProps & {
+  isBusy?: boolean;
+};
+
+const SpeciesDeliverableView = (props: Props): JSX.Element => {
+  const { ...viewProps }: ViewProps = props;
+  const dispatch = useAppDispatch();
+  const { isMobile } = useDeviceInfo();
+  const { activeLocale } = useLocalization();
+  const theme = useTheme();
+  const { currentParticipantProject } = useParticipantData();
+  const participantProjectSpecies = useAppSelector(
+    selectParticipantProjectSpeciesListRequest(currentParticipantProject?.id || -1)
+  );
```

Comment:
Any interest in providing `currentParticipantProjectSpecies` from the `ParticipantProvider`? Not sure if it makes sense since this is currently the only view consuming that data.

Reply:
Not necessary for now I guess